### ISSUE TITLE
Option to allow best-effort for training with few samples

### ISSUE THREAD

**Issue by YiannisGkoufas** [mljar/mljar-supervised#453]
Hi there!

I tried to train a model with a few number of samples and I get:
```
skewtest is not valid with less than 8 samples; 6 samples were given.
```
I realize that the skewtest is needed, but would it be possible to have an option to have a "best-effort" functionality?

**Comment by pplonski** [comment]
Hi @YiannisGkoufas,

We can try ... Does all work as expected if you increase the number of samples to 8?

**Comment by YiannisGkoufas** [comment]
Yes like a breeze :) That's why I think it would be a feature, it works as expected
